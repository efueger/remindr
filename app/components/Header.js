import React from 'react';
import {
  StyleSheet,
  Text,
  View,
} from 'react-native';

const styles = StyleSheet.create({
  container: {
    borderColor: '#d9dbdb',
    borderBottomWidth: 1,
  },
  title: {
    fontSize: 29,
    fontWeight: '300',
    paddingTop: 20,
    paddingBottom: 20,
    paddingLeft: 32,
    paddingRight: 32,
  },
});

export default function Header() {
  return (
    <View style={styles.container}>
      <Text style={styles.title}>Remindr</Text>
    </View>
  );
}
