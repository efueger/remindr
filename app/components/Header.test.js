import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

it('renders the header component', () => {
  const component = renderer.create(
    <Header />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
