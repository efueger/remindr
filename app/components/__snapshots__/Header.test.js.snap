exports[`test renders the header component 1`] = `
<View
  style={
    Object {
      "borderBottomWidth": 1,
      "borderColor": "#d9dbdb"
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 29,
        "fontWeight": "300",
        "paddingBottom": 20,
        "paddingLeft": 32,
        "paddingRight": 32,
        "paddingTop": 20
      }
    }>
    Remindr
  </Text>
</View>
`;
